docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -41,13 +41,18 @@
    "metadata": {},
```

Comment:
<p>... provide any callback function...</p><p>I'd also  and describe the arguments (iteration number, etc.), what's already in the docstring of <code>default_progress_fn</code> and when mentioning the <code>NamedTuple</code>, mention the {func}<code>~ott.solvers.linear.sinkhorn.SinkhornOutput.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>